src/viur/core/bones/base.py

```diff
@@ -294,6 +300,10 @@ def __init__(
 
         self.compute = compute
 
+    def __set_name__(self, owner: t.Any, name: str) -> None:
```

Comment:
```suggestion
    def __set_name__(self, owner: `Skeleton`, name: str) -> None:
```

Reply:
Okay, that obviously depends on your point of view. I'm taking the general case first: A bone can be packed into any python object. In the ViUR context, of course, this will usually only be a skeleton.